### PR TITLE
make sure event messages end up one one line and include counts

### PR DESCRIPTION
**Pull request by deads2k**
/assign @smarterclayton

### CODE REVIEW

**Comment by deads2k** [comment]
@bparees this is highlighting a problem with config observation for images or builds

`Jan 14 20:31:38.637 I ns=openshift-kube-apiserver-operator deployment=openshift-kube-apiserver-operator Allowed registries for import changed to [] count(91)`

**Comment by deads2k** [comment]
comments were addressed. tagging

**Comment by bparees** [comment]
@deads2k this was already reported by @sjenning, and from what I see it's only an issue w/ the logic that detects whether the value has changed or not (hence whether or not to record an event).

In the interest of expediency I have removed the event recording logic:
https://github.com/openshift/cluster-kube-apiserver-operator/pull/198

If you can see something obviously wrong w/ the fix @mfojtik attempted on it earlier, we can revisit:
https://github.com/openshift/cluster-kube-apiserver-operator/pull/188

File: pkg/monitor/cmd.go [hunk 1]
```diff
@@ -60,7 +61,8 @@ func (opt *Options) Run() error {
 					if !event.From.Equal(event.To) {
 						continue
 					}
-					fmt.Fprintln(opt.Out, event.String())
+					// make sure the message ends up on a single line.  Something about the way we collect logs wants this.
```
Comment (soltysh) [inline pkg/monitor/cmd.go]: Can we fix that something, this is too volatile, imho.

**Pull request merged**